index.js

```diff
@@ -485,13 +485,53 @@ const wealthsimple = {
    * Information about a security on the WealthSimple Trade Platform.
    *
    * @param {*} tokens The access and refresh tokens returned by a successful login.
-   * @param {*} ticker The security symbol
+   * @param {string|object} ticker The security symbol. An exchange may be added as a suffix, separated from the symbol with a colon, for example: AAPL:NASDAQ, ENB:TSX
+   * @param {string} ticker.symbol The security symbol.
+   * @param {string} [ticker.exchange] (optional) the exchange the security trades in
+   * @param {string} [ticker.id] (optional) The internal WealthSimple Trade security ID
    */
   getSecurity: function () {
     var _getSecurity = _asyncToGenerator(function* (tokens, ticker) {
-      return handleRequest(_endpoints.default.SECURITY, {
-        ticker
+      if (typeof ticker === 'string') {
+        ticker = {
+          symbol: ticker
+        };
+        let tickerParts = ticker.symbol.split(':');
+
+        if (tickerParts.length > 2) {
+          return Promise.reject({
+            reason: `Illegal ticker: ${ticker.symbol}`
+          });
+        }
+
+        ticker.exchange = tickerParts[1];
```

Comment:
I haven't had the chance to test the code yet, but this seems like it would require exchanges to always be postfixed when the ticker is specified as a string. It should be optional as we've discussed.

Reply:
If the ticker is not postfixed, this sets `ticker.exchange` to `undefined`, which is falsey and skips the exchange logic.

Reply:
Gotcha. Thanks for the clarification.